### PR TITLE
setup_environment: parallelize bitcoind build

### PR DESCRIPTION
**Pull request by brunoerg**
See https://github.com/bitcoin-core/HWI/pull/752#discussion_r1743768282

### CODE REVIEW

**Comment by brunoerg** [comment]
cc: @hebasto

File: test/setup_environment.sh [hunk 1]
```diff
@@ -449,5 +449,5 @@ if [[ -n ${build_bitcoind} ]]; then
 
     # Do the build
     cmake -B build --toolchain depends/x86_64-pc-linux-gnu/toolchain.cmake -DBUILD_TESTS=OFF -DBUILD_BENCH=OFF
-    cmake --build build --target bitcoind
+    cmake --build build -j $(nproc) --target bitcoind
```
Comment (hebasto) [inline test/setup_environment.sh:452]: Same for building depends a few line above?

Comment (brunoerg) [reply test/setup_environment.sh:452]: Done

**Comment by brunoerg** [comment]
Force-pushed addressing https://github.com/bitcoin-core/HWI/pull/755#discussion_r1748023325

**Review by hebasto — APPROVED** [review]
ACK 1dbc4fd8833b415bf97976070bbdd940f25594a0.

**Pull request merged**